### PR TITLE
IAE-24147 added drawer content for the sub header

### PR DESCRIPTION
**Pull request by yerramshilpa**
## Type of Change (Select One and Apply Github Label)

- [ ] Bug fix (non-breaking change which fixes an issue) -> Apply bugfix label
- [x] New feature (non-breaking change which adds functionality) -> Apply enhancement label
- [ ] Breaking change (fix or feature that would cause existing functionality to change) -> Apply breaking label

## Which browsers have you tested?
- [ ] Internet Explorer 11
- [ ] Edge
- [x] Chrome
- [ ] Firefox
- [x] Safari

## Checklist:

- [ ] I have read the (https://github.comv/GSA/sam-ui-elements/blob/CONTRIBUTING.md)[CONTRIBUTING.md] document.
- [ ] My code passes the automated linter.
- [ ] This code has been reviewed by another team member and passes the reviewer checklist found in (https://github.comv/GSA/sam-ui-elements/blob/CONTRIBUTING.md)[CONTRIBUTING.md]
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] My code is 508 compliant as tested by AMP and JAWS
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/component-sample/src/lib/feature/subheader-sample/subheader-sample.component.html [hunk 1]
```diff
@@ -1,23 +1,45 @@
...
-</sds-subheader>
+    <!-- =============== ACTIONS BUTTONS =============== -->
+    <sds-subheader-actions [model]="subheader.actions" (clicks)="log($event)"></sds-subheader-actions>
+    <!-- =============== DRAWER CONTENT =============== -->
+    <sds-subheader-drawer [drawerContentTemplate]="temp"></sds-subheader-drawer>
+   
+  </sds-subheader>
+
+  <ng-template #temp>
+    <div style="margin-top:-15px; background-color: #eff6fb; padding:0 10px;">
```
Comment (davereed) [inline libs/component-sample/src/lib/feature/subheader-sample/subheader-sample.component.html]: Remove margin (the content is adding this from the first P tag), try adding a heading instead.

Use classes: 
bg-accent-cool-lighter
padding-1

File: libs/packages/layouts/src/lib/feature/subheader/subheader-drawer.component.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<!-- <sds-drawer (isDrawerOpen)="isDrawerOpen.emit($event)" [contentTemplate]="contentTemplate"></sds-drawer> -->
+
+<button (click)="onDrawerOpenClose($event)"  class="usa-button sds-button--circle">
+    <fa-icon [fixedWidth]="true" [icon]="isOpen ?['fas', 'arrow-down']: ['fas', 'arrow-up']"></fa-icon>
```
Comment (davereed) [inline libs/packages/layouts/src/lib/feature/subheader/subheader-drawer.component.html]: Reverse the arrows:
Down arrow when closed
Up arrow when open

File: libs/component-sample/src/lib/feature/subheader-sample/subheader-sample.component.html [hunk 3]
```diff
@@ -1,23 +1,45 @@
...
-  <!-- =============== ACTIONS BUTTONS =============== -->
-  <sds-subheader-actions
-    [model]="subheader.actions"
-    (clicks)="log($event)"
-  ></sds-subheader-actions>
-</sds-subheader>
+    <!-- =============== ACTIONS BUTTONS =============== -->
+    <sds-subheader-actions [model]="subheader.actions" (clicks)="log($event)"></sds-subheader-actions>
+    <!-- =============== DRAWER CONTENT =============== -->
+    <sds-subheader-drawer [drawerContentTemplate]="temp"></sds-subheader-drawer>
```
Comment (davereed) [inline libs/component-sample/src/lib/feature/subheader-sample/subheader-sample.component.html]: Can we rename temp to template (throughout the PR)? We don't want to confuse temporary with template.

File: libs/component-sample/src/lib/feature/subheader-sample/subheader-sample.component.html (hunk 1 above)
Comment (yerramshilpa) [reply libs/component-sample/src/lib/feature/subheader-sample/subheader-sample.component.html]: Updated

File: libs/packages/layouts/src/lib/feature/subheader/subheader-drawer.component.html (hunk 2 above)
Comment (yerramshilpa) [reply libs/packages/layouts/src/lib/feature/subheader/subheader-drawer.component.html]: reversed the icons

File: libs/component-sample/src/lib/feature/subheader-sample/subheader-sample.component.html (hunk 3 above)
Comment (yerramshilpa) [reply libs/component-sample/src/lib/feature/subheader-sample/subheader-sample.component.html]: changed to contentTemplate

**Pull request merged**